### PR TITLE
adding validation input

### PR DESCRIPTION
**Pull request by TuyizeeAnastase**
What does this PR do?
#validating input contact form

Description of task to be completed?
#validate input name and email value empty
#validate email as formated

### CODE REVIEW

**Comment by TuyizeeAnastase** [comment]
trello story
#zcJf8Vny

**Comment by TuyizeeAnastase** [comment]
Can you my PR on validating input @Ntwali-Josue

File: script.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    var name=document.getElementById("Name").value;
+    var email=document.getElementById("Email").value;
+    var message=document.getElementById("message").value;
+
+    if(name==""){
+       alert("Please enter Your Names");
+       return false;
+    }
+    else if(email==""){
+        alert("Please enter Your user Email");
```
Comment (Ntwali-Josue) [inline script.js:11]: What if someone puts an email without an @ or .com?! or put anything.. i think your email input will accpet them which isn't a good approach. i urge you to work on this!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+         return false;
+     }
```
Comment (Ntwali-Josue) [inline script.js:13]: What if someone puts an email without an @ or .com?! or put anything.. i think your email input will accpet them which isn't a good approach. i urge you to work on this!

Comment (TuyizeeAnastase) [reply script.js:13]: I done it though  my input html and type="Email" automatically validate it without imputing conditions in js

**Pull request merged**